packages/eas-cli/src/metadata/context.ts

```diff
@@ -41,8 +42,8 @@ export type MetadataAppStoreAuthentication = {
  */
 export async function createMetadataContextAsync(params: {
   projectDir: string;
-  credentialsCtx: CredentialsContext;
-  exp: ExpoConfig;
+  credentialsCtx?: CredentialsContext;
+  exp?: ExpoConfig;
```

Comment:
I think this can be left as non-nullable and the fallback below be removed?

Reply:
Ah, cool, yeah splitting it up would work as well.

Reply:
Done!